tools/map_migrations/3739_asteroid_turfs.txt

```diff
@@ -0,0 +1,9 @@
+/turf/simulated/floor/airless/lava/@SUBTYPES : /turf/exterior/lava/@SUBTYPES{@OLD}
+/turf/simulated/floor/ice/@SUBTYPES : /turf/exterior/ice/@SUBTYPES{@OLD}
+/turf/simulated/floor/snow/@SUBTYPES : /turf/exterior/snow/@SUBTYPES{@OLD}
+/turf/simulated/floor/airless/ceiling/@SUBTYPES : /turf/simulated/floor/airless/@SUBTYPES{@OLD}
+/turf/simulated/floor/beach/sand/@SUBTYPES : /turf/exterior/sand/@SUBTYPES{@OLD}
+/turf/simulated/floor/beach/sand/desert/@SUBTYPES : /turf/exterior/sand/@SUBTYPES{@OLD}
```

Comment:
This line is redundant with the above but it won't hurt anything. Also, noting for myself that the desert sand texture is the more complex one the exterior sand turf uses anyway, so the only loss here is the base sand type that looks kind of ugly.